scripts/hooks/pre-commit

```diff
@@ -6,10 +6,11 @@
 
 FAST_LINT=true
 
-cd server
+export PATH=$PATH:~/.nvm/versions/node/v18.16.0/bin
```

Comment:
this is how we tell the git hook where `pnpm` is so it stops whining about it not existing.